Makefile

```diff
@@ -1,4 +1,3 @@
-CC := gcc
```

Comment:
Rather than removing sections of the Makefile, can you change this so that the CC and TEST-CCFLAGS are optionally included for your fuzz target?